packages/@ember/-internals/glimmer/lib/helpers/custom.ts

```diff
@@ -430,6 +415,8 @@ export function invokeHelper(
     manager
   );
 
+  assert('Invoke helper does not support internal helpers yet', !isInternalHelper(manager));
```

Comment:
This seems like a new addition? Should we consider backporting this?

Reply:
This is a new addition, it is to allow us to associate internal helpers directly with a function via `setHelperManager`. This way we can make all of the built-in helpers work in strict mode, when you import them.

Currently, those internal helpers get resolved above, and we don't even associate any internal helpers directly with `setHelperManager`, so this assertion should never be triggered. I added it to satisfy the type constraints.